PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1769,7 +1769,7 @@ bool FindReplaceDlg::processFindNext(const TCHAR *txt2find, const FindOption *op
 		}
 
 		if (posFind == -1)

```

Comment:
In the preceding files, posFind was compared to -1 or -2, which was modified to compare to 0.  This compares only to -1.  Is this an intentional comparison to a specific error return code?